hooks/useFavourite.ts

```diff
@@ -0,0 +1,24 @@
+import {useState} from "react";
+import {getDataFromStorage, setDataToStorage} from "../utils/getDataFromStorage";
+
+export const useFavourite = (date: string) => {
+  const favouritePictures = getDataFromStorage("favPics");
```

Comment:
I would suggest to move `favPics` literall to a constant, because it's already used in three places.